### DISCUSSION TITLE
Upload failed

### DISCUSSION THREAD

**Discussion by hsx02** [ROM-builders/temporary#9825] [General]
https://cirrus-ci.com/task/4503781856509952?logs=upload#L13

**Comment by Apon77** [comment]
<img width="755" alt="image" src="https://user-images.githubusercontent.com/61594531/169325951-cb956f4b-3438-444f-8fa9-7f917d5b68eb.png">

**Comment by Apon77** [comment]
> https://cirrus-ci.com/task/4503781856509952?logs=upload#L13

its because your custom rclone command. your fault obviously
